src/coreclr/src/inc/eventtracebase.h

```diff
@@ -1246,7 +1246,7 @@ namespace ETW
 #define ETWLoaderStaticLoad 0 // Static reference load
 #define ETWLoaderDynamicLoad 1 // Dynamic assembly load
 
-#if defined(FEATURE_EVENT_TRACE) && !defined(TARGET_UNIX)
+#if defined(FEATURE_EVENT_TRACE) && !defined(HOST_UNIX)
```

Comment:
Make matching changes in the .cpp file for consistency?

Reply:
(Or it may be better to not include this whole file in the DAC build if possible.)

Reply:
I tried removing the file form the build today.  It wasn't trivial.  It broke a lot of other files which have taken dependencies on things this file defines.  I am taking the matching changes in .cpp approach.